lib/l10n/intl_en.arb

```diff
@@ -907,6 +907,38 @@
   "@demoSimpleDialogDescription": {
     "description": "Description for the simple dialog component demo."
   },
+  "demoDividerTitle": "Divider",
+  "@demoDividerTitle": {
+    "description": "Title for the divider component demo."
+  },
+  "demoDividerSubtitle": "A divider is a thin line that groups content in lists and layouts.",
+  "@demoDividerSubtitle": {
+    "description": "Subtitle for the divider component demo."
+  },
+  "demoDividerDescription": "Dividers can be used in lists, drawers, and elsewhere to separate content.",
+  "@demoDividerDescription": {
+    "description": "Description for the divider component demo."
+  },
+  "demoDividerAbove": "Above",
```

Comment:
@rami-a , Should I remove these text for container ?
Since It won't affect demo and will also reduce translation work.

Reply:
Actually yes, this is a good idea. I say remove the text, then you can even remove the `_BuildContainer` helper, and simply use Container classes in line in the demo to simplify the code.